### PR TITLE
Ltd 979 query status api endpoint

### PR DESCRIPTION
**Pull request by dgnball**
Add two new APIs routes and a sample integration test that makes use of them. The purpose of this is to allow an integration test to check that API calls made to lite-hmrc result in an email being sent to HMRC.

### CODE REVIEW

**Review by saruniitr — CHANGES_REQUESTED** [review]

File: system_tests/expected_mail_body.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+3\trader\\GB123456789000\20200602\20220602\Organisation\might\248 James Key Apt. 515\Apt. 942\West Ashleyton\Farnborough\GU40 2LX
```
[the remark this answers is not in the document]
Comment (dgnball) [reply system_tests/expected_mail_body.txt]: Ok, I am testing lines 1 and 2 now.

File: system_tests/hmrc_system_test.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    clear_stmp_mailbox()
+    put("/mail/set-all-to-reply-sent/")
+    with open("licence_payload_file.json") as f:
+        data = json.load(f)
+    data["licence"]["id"] = str(uuid.uuid4())
+    data["licence"]["reference"] = f"GBSIEL/2020/{''.join(random.sample(string.ascii_lowercase, 7))}/P"
+    post("/mail/update-licence/", data=data)
+    put("/mail/send-licence-updates-to-hmrc/")
+    body = get_smtp_body().replace("\r", "")
+    # Remove the first and second line as it contains data that changes every time
```
[the remark this answers is not in the document]
Comment (dgnball) [reply system_tests/hmrc_system_test.py]: Done

[the remark this answers is not in the document]
Comment (dgnball) [reply system_tests/hmrc_system_test.py]: Done

File: system_tests/hmrc_system_test.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        requests.delete(f"http://localhost:8025/api/v1/messages/{id}")
+
+
+def get_smtp_body():
+    response = requests.get("http://localhost:8025/api/v2/messages")
+    print(response)
+    return response.json()["items"][0]["MIME"]["Parts"][1]["Body"]
+
+
+def test_send_license():
```
[the remark this answers is not in the document]
Comment (dgnball) [reply system_tests/hmrc_system_test.py]: Done

File: mail/urls.py [hunk 4]
```diff
@@ -9,5 +9,6 @@
     # The endpoints below are for testing purposes; they can be enabled to manually trigger and debug Background Tasks
     # path("manage-inbox/", ManageInbox.as_view(), name="manage_inbox"),
     # path("send-usage-updates-to-lite-api/", SendUsageUpdatesToLiteApi.as_view(), name="send_updates_to_lite_api"),
-    # path("send-licence-updates-to-hmrc/", SendLicenceUpdatesToHmrc.as_view(), name="send_updates_to_hmrc"),
+    path("send-licence-updates-to-hmrc/", views.SendLicenceUpdatesToHmrc.as_view(), name="send_updates_to_hmrc"),
+    path("set-all-to-reply-sent/", views.SetAllToReplySent.as_view(), name="set_all_to_reply_sent"),
```
Comment (saruniitr) [inline mail/urls.py:13]: Please include them under `DEBUG` flag which is `False` in `prod` as these endpoints should not be available in `prod`

**Review by saruniitr — CHANGES_REQUESTED** [review]
one last change otherwise looks good to me.

File: mail/urls.py (hunk 4 above)
Comment (dgnball) [reply mail/urls.py]: Done

File: mail/views.py [hunk 5]
```diff
@@ -89,22 +89,25 @@ def post(self, request):
 
 
 class ManageInbox(APIView):
-    def get(self, request):
```
[the remark this answers is not in the document]
Comment (dgnball) [reply mail/views.py:92]: Done

File: mail/urls.py [hunk 6]
```diff
@@ -1,13 +1,18 @@
 from django.urls import path
 
+from conf import settings
```
Comment (saruniitr) [inline mail/urls.py:3]: This should be `from django.conf import settings`
`settings` is actually a `dict`, it shouldn't be imported as module

**Review by saruniitr — CHANGES_REQUESTED** [review]

File: mail/urls.py (hunk 6 above)
Comment (dgnball) [reply mail/urls.py:3]: If I do this using django.conf, settings doesn't pick up what I put into the .env file (always set to False)

Comment (dgnball) [reply mail/urls.py]: Decided not to preclude test URLs with DEBUG

**Review by saruniitr — APPROVED** [review]

**Pull request merged**